.gcloud/properties

```diff
@@ -1,3 +0,0 @@
-[core]
```

Comment:
This used to tell the `gcloud` CLI tool used for deploys etc what the project name was on App Engine.  Google deprecated this, so now we have to put the project name in the Makefile and pass it to the deploy command directly.